src/openforms/validations/api/serializers.py

```diff
@@ -9,6 +9,10 @@ class ValidationInputSerializer(serializers.Serializer):
     )
 
 
+class ValidationRequestParamSerializer(serializers.Serializer):
+    component = serializers.CharField()
```

Comment:
typically you'd use a filter set/backend for this instead of a serializer, see https://django-filter.readthedocs.io/en/stable/guide/rest_framework.html

This is then also properly picked up by the API schema generation.

Reply:
The library requires models, so this suggestion is plain wrong